QUICKSTART.md

```diff
@@ -0,0 +1,165 @@
+# NgRX Auto-Entity
+
+Automatic entities for [@ngrx](https://github.com/ngrx/platform)! With this library, you can take the boiler off your plate, and
+get back to business!
+
+# Quick Start
+
+One of the primary goals with Auto-Entity is to provide a rapid development platform
+with minimal boilerplate requirements and easy setup. For those developers who wish
+to get rolling as quickly as possible, this guide should get you going fast!
+
+## Installation
+
+Install @briebug/ngrx-auto-entity from npm:
+
+`npm install @briebug/ngrx-auto-entity` or `yarn add @briebug/ngrx-auto-entity`
+
+## Setup
+
+### 1: App Module
+
+Add the `NgrxAutoEntityModule` to your app module:
+
+```typescript
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { NgrxAutoEntityModule } from '@briebug/ngrx-auto-entity';
+
+import { AppComponent } from './app.component';
+
+@NgModule({
+  declarations: [AppComponent],
+  imports: [BrowserModule, NgrxAutoEntityModule],
+  bootstrap: [AppComponent]
+})
+export class AppModule {}
+```
+
+### 2: Meta Reducer
+
+Add the autoEntityMetaReducer to your meta reducers for your application or
+feature:
+
+```typescript
+import { routerReducer } from '@ngrx/router-store';
+import { ActionReducerMap, MetaReducer } from '@ngrx/store';
+import { autoEntityMetaReducer } from '@briebug/ngrx-auto-entity';
+import { storeFreeze } from 'ngrx-store-freeze';
+// ...
+import { environment } from '../../environments/environment';
+
+export interface IAppState {
+  // ...
+}
+
+export const appReducer: ActionReducerMap<IAppState> = {
+  // ...
+};
+
+export const appMetaReducers: Array<MetaReducer<IAppState>> = !environment.production
+  ? [autoEntityMetaReducer, storeFreeze]
+  : [autoEntityMetaReducer];
+```
+
+### 3: App State
+
+Add default `EntityEffects` to your effects in your state module and reference
+your meta reducers in the store:
+
+```typescript
+// ...
+import { EntityEffects } from '@briebug/ngrx-auto-entity';
+import { appMetaReducers, appReducer } from './app.reducer';
+// ...
+
+@NgModule({
+  imports: [
+    // ...
+    StoreModule.forRoot(appReducer, { metaReducers: appMetaReducers }),
+    EffectsModule.forRoot([EntityEffects /* ... */])
+    // ...
+  ]
+})
+export class StateModule {
+  // ...
+}
+```
+
+### 4: Entity State
+
```

Comment:
Should we have the instructions on setting up a Model class here?

Reply:
Added example model and service code in step #5